specification/maps/data-plane/Search/preview/2023-04-01-preview/examples/AutosuggestAddress.json

```diff
@@ -0,0 +1,46 @@
+{
+  "parameters": {
```

Comment:
Please rename the following parameters to align with the Swagger file:
- `userLocation` -> `location`
- `resultType` -> `resultTypes`

Also, the values for `query` (`1 Microsoft Way, Redmond, WA`) and `resultTypes` (`Address`) must also be changed to align with response and examples in the PM spec.